src/layman/upgrade/upgrade_v2_0.py

```diff
@@ -347,19 +347,19 @@ def migrate_layers():
                         dst_filepath = os.path.join(gdal_dir, filename)
                         shutil.move(src_filepath, dst_filepath)
 
-                filenames = set(
-                    os.path.splitext(os.path.basename(filename))[0] for filename in os.listdir(gdal_old_dir) if filename.endswith('.tif'))
-                for old_file_name in filenames:
-                    new_file_name = layer_uuid if image_mosaic is None else old_file_name
-                    src_filepath = os.path.join(gdal_old_dir, f"{old_file_name}.tif")
-                    dst_filepath = os.path.join(gdal_dir, f"{new_file_name}.tif")
-                    shutil.move(src_filepath, dst_filepath)
-
-                    src_filepath = os.path.join(gdal_old_dir, f"{old_file_name}.tif.aux.xml")
-                    dst_filepath = os.path.join(gdal_dir, f"{new_file_name}.tif.aux.xml")
-                    if os.path.exists(src_filepath):
+                all_filenames = set(
+                    os.path.basename(filename) for filename in os.listdir(gdal_old_dir))
+                tif_filenames = set(
+                    os.path.splitext(filename)[0] for filename in all_filenames if filename.endswith('.tif'))
+                main_filenames = set(tif_filename for tif_filename in tif_filenames if tif_filename == layername or image_mosaic)
+                for src_filename in main_filenames:
+                    file_to_move = {filename for filename in all_filenames if filename.startswith(f'{src_filename}.tif')}
```

Comment:
Just to mention, this will move all files starting with `<layername>.tif`, whereas my intention was to move all files starting with `<layername>` (or `<any_tif_file_without_extention>`). However, if it works on 157, I guess there is big chance this will work also on other machines.